tools/cldr-code/src/main/java/com/ibm/icu/dev/test/TestFmwk.java

```diff
@@ -846,18 +890,28 @@ public void msg(String message, int level, boolean incCount, boolean newln) {
         params.msg(message, level, incCount, newln);
     }
 
-    static final String ICU_TRAC_URL = "http://bugs.icu-project.org/trac/ticket/";
-    static final String CLDR_TRAC_URL = "http://unicode.org/cldr/trac/ticket/";
-    static final String CLDR_TICKET_PREFIX = "cldrbug:";
+    static final Pattern ICU_TICKET_PATTERN = PatternCache.get(
+        "(?i)(?:icu-)?(\\d+)"
+    );
+    static final Pattern CLDR_TICKET_PATTERN = PatternCache.get(
+        "(?i)cldr(?:bug:)?(?:-)(\\d+)"
```

Comment:
Doesn't this require a dash even after `cldrbug:` ?

Reply:
fixed